aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -239,14 +239,20 @@ CREATE_UNARY_UFUNC_FUNCTIONAL(isinf);
 
 // pow function for Sparse CSR, since square is composite - this will also enable square for Sparse CSR
 Tensor& pow_sparse_csr_scalar_out(const Tensor& self, const Scalar& exponent, Tensor& result) {
+  TORCH_CHECK(!(exponent.isBoolean() && exponent.equal(false)), "Exponent as False is not supported for Sparse CSR Layout.");
+  TORCH_CHECK(!exponent.equal<int>(0), "Exponent as 0 is not supported for Sparse CSR Layout.");
   return unary_op_out<Tensor& (*)(const Tensor&, const Scalar&, Tensor&)>(&at::pow_outf, self, result, exponent);
 }
 
 Tensor pow_sparse_csr_scalar(const Tensor& self, const Scalar& exponent) {
+  TORCH_CHECK(!(exponent.isBoolean() && exponent.equal(false)), "Exponent as False is not supported for Sparse CSR Layout.");
+  TORCH_CHECK(!exponent.equal<int>(0), "Exponent as 0 is not supported for Sparse CSR Layout.");
```

Comment:
`0**-x` is `inf` not zero.
```suggestion
  TORCH_CHECK(exponent.to<double>() > 0.0, "Exponent must be greater than 0 for Sparse CSR Layout.");
```

Reply:
Thanks!